torch/testing/_internal/common_cuda.py

```diff
@@ -15,7 +24,7 @@
 CUDA_ALREADY_INITIALIZED_ON_IMPORT = torch.cuda.is_initialized()
 
 
-TEST_MULTIGPU = TEST_CUDA and torch.cuda.device_count() >= 2
+TEST_MULTIGPU = (TEST_CUDA or TEST_HPU or TEST_XPU) and torch.accelerator.device_count() >= 2
```

Comment:
Overall LGTM. `torch/testing/_internal/common_cuda.py` is the CUDA-specific.
I think we could add a new `TEST_MULTIACCELERATOR` maybe in `torch/testing/_internal/common_utils.py`